plugins/source/aws/resources/plugin/plugin_test.go

```diff
@@ -15,12 +16,15 @@ func TestAWS(t *testing.T) {
 	}
 }
 
-// This test ensures that all tables have a unique description.
-func TestAWSTableDescriptions(t *testing.T) {
+// This test ensures that all tables have proper name and description.
+func TestAWSTables(t *testing.T) {
 	descriptions := make(map[string]string)
 	tables := getTables().FlattenTables()
 	for _, table := range tables {
-		if ignoreTable(table.Name) {
+		if !ignorePluralName(table.Name) && !strings.HasSuffix(table.Name, "s") {
```

Comment:
We might have discussed this before, but should we use https://github.com/gertd/go-pluralize/tree/master for this? It covers more use cases